web/geosearch/datapunt_geosearch/blueprints/search.py

```diff
@@ -15,6 +19,33 @@
 
 search = Blueprint('search', __name__)
 
+_logger = logging.getLogger(__name__)
+
+
+def retry_on_psycopg2_error(name):
+    """
+    Decorator that retries 3 times after Postgres error, in particular if
+    the connection was not valid anymore because the database was restarted
+    """
+    def decorator_retry(func):
+        @functools.wraps(func)
+        def wrapper_retry(*args, **kwargs):
+            retry = 3
+            while retry > 0:
```

Comment:
while not result or retry < 3

Reply:
Het zou moeten zijn  : while not result and retry > 0 
Maar ik moest er zolang over nadenken dat ik het duidelijker vind zoals het nu is